src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -64,6 +64,21 @@ public static void Verify(string expectedOperationTree, string actualOperationTr
 
         #region Logging helpers
 
+        private void LogPatternPropertiesAndNewLine(IPatternOperation operation)
+        {
+            LogPatternProperties(operation);
+            LogString(")");
+            LogNewLine();
+        }
+
+        private void LogPatternProperties(IPatternOperation operation)
+        {
+            LogString(" (");
+            // Kind
+            LogString($"{nameof(OperationKind)}.{operation.Kind}");
+            LogType(operation.InputType, $", {nameof(operation.InputType)}");
```

Comment:
Missing many features from `LogCommonPropertiesAndNewLine`. Consider removing this and using the existing functions.